tests/general/InvalidLinkBearTest.py

```diff
@@ -126,6 +126,7 @@ def test_run(self):
         http://httpbin.org/status/410
         http://httpbin.org/status/500
         http://httpbin.org/status/503
+        http://httpbin.org/status/200 http://httpbin.org/status/404
```

Comment:
Ugh, actually the logic of this test wont help verify that there were multiple results. Create a new test method please.